tests/unittests/CMakeLists.txt

```diff
@@ -124,6 +124,18 @@ target_link_libraries(graphOptzTest
                         testMain)
 add_glow_test(graphOptzTest ${GLOW_BINARY_DIR}/tests/graphOptzTest)
 
+add_executable(graphSchedulerTest
+	       graphSchedulerTest.cpp)
+target_link_libraries(graphSchedulerTest
+                      PRIVATE
+                        Graph
+                        IR
+                        gtest
+                        testMain)
+target_include_directories(graphSchedulerTest PUBLIC ${CMAKE_SOURCE_DIR}/lib/IR)
```

Comment:
not sure if this needs to be `PUBLIC`. As those headers only used in the `graphSchedulerTest.cpp`, `PRIVATE` should be enough.

Reply:
Done.